cernopendata/modules/records/utils.py

```diff
@@ -162,8 +162,8 @@ def record_file_page(pid, record, page=1, **kwargs):
     except Exception:
         items_per_page = 5
 
-    _files = record.files
-    index_files = record._file_indices
+    _files = record.get("files", [])
```

Comment:
I think the problem is https://github.com/cernopendata/cernopendata-portal/blob/853b55af0554605913fa1005afc56142a9ed2ba6/cernopendata/config.py#L205 we should point to the new API class i.e. [RecordFilesWithIndex](https://github.com/cernopendata/cernopendata-portal/blob/853b55af0554605913fa1005afc56142a9ed2ba6/cernopendata/api.py#L47)

Reply:
Thanks!  Let me check that